src/Button/Button.less

```diff
@@ -0,0 +1,89 @@
+a, button {
+  &.Button {
+    margin: 0 8px;
+    background: #ffffff;
+    border: 0;
+    border-top-right-radius: 3px;
+    border-top-left-radius: 3px;
+    cursor: pointer;
+    text-decoration: none;
+
+    transition: background-color 0.25s ease-out, border-color 0.25s ease-out, color 0.25s ease-out;
+
+    &[disabled]{
+      cursor: not-allowed;
+      color: #4B4B59;
+      background-color: #D7D9D9;
+      border-bottom: 3px solid #A4A6A6;
+    }
+
+    &:hover,
+    &:focus,
+    &:active{
+      outline: 0;
+    }
+  }
+
+  &.Button--primary {
+    background-color: #4274F6;
+    color: #FAFCFC;
+    border-bottom: 3px solid #345CC4;
+
+    &:hover,
+    &:focus,
+    &:active {
+      background-color: #3B68DD;
+      color: #FAFCFC;
+      border-bottom: 3px solid #2E51AC;
+    }
+  }
+
+  &.Button--secondary {
+    background-color: #FAFCFC;
+    color: #4274F6;
+    border: 1px solid #5481F6;
+    border-bottom: 3px solid #677FF7;
+
+    &:hover,
+    &:focus,
+    &:active {
+      background-color: #FAFCFC;
+      color: #345CC4;
+      border: 1px solid #345CC4;
+      border-bottom: 3px solid #345CC4;
+    }
+  }
+
+  &.Button--destructive {
+    background-color: #EB3B49;
+    color: #FAFCFC;
+    border-bottom: 3px solid #BC2F3A;
+
+    &:hover,
+    &:focus,
+    &:active {
+      background-color: #D33541;
+      color: #FAFCFC;
+      border-bottom: 3px solid #A42933;
+    }
+  }
+
+  &.Button--small {
+    padding: 10px 15px 5px 15px;
+    font-size: 10px;
+    font-weight: semibold;
```

Comment:
don't think this is a valid [font-weight](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight), probably want `600`